fire/core.py

```diff
@@ -141,7 +146,8 @@ def Fire(component=None, command=None, name=None):
     result = component_trace.GetResult()
     print(
         helputils.HelpString(result, component_trace, component_trace.verbose))
-    return None
+    # ape argparse by exiting out quickly
+    raise FireExit(0, None)
```

Comment:
Exiting quickly w/ code=0 sgtm. Let's still include the component_trace in the FireExit.